actions/post_actions.jsx

```diff
@@ -317,6 +317,37 @@ export function hideEditPostModal() {
     };
 }
 
+export function deleteCombinedPost(combinedPost) {
+    return async (doDispatch, doGetState) => {
+        const state = doGetState();
+
+        combinedPost.system_post_ids.forEach((systemPostId) => {
```

Comment:
`deleteCombinedPost` - I suggests not to move this function to redux, since it has sequence and batch in dispatching actions. This has similar implementation to what we're doing with `deleteAndRemovePost` but very specific to combined post.

Reply:
I don't understand. Isn't `deleteCombinedPost` similar to calling `deleteAndRemovePost` on each of the system posts and then once more on the combined post? I'm not sure what part of that would be specific to the web app